src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2249,24 +2158,7 @@ because xbuild doesn't support framework reference assemblies.
   >
     <Output TaskParameter="Processed" ItemName="_ProcessedCustomViews" />	
   </ConvertCustomView>
-  <Delete Files="@(_ProcessedCustomViews->'$(_AndroidLibraryFlatArchivesDirectory)%(Hash).stamp')"
-    Condition=" '$(AndroidUseAapt2)' == 'True' And '@(_ProcessedCustomViews)' != '' "
-  />
-  <Aapt2Compile
-    Condition=" '$(AndroidUseAapt2)' == 'True' And '@(_ProcessedCustomViews)' != '' "
-    ContinueOnError="$(DesignTimeBuild)"
-    ResourceDirectories="@(_LibraryResourceHashDirectories);$(MonoAndroidResDirIntermediate)"
-    ExplicitCrunch="$(AndroidExplicitCrunch)"
-    ExtraArgs="$(AndroidAapt2CompileExtraArgs)"
-    FlatArchivesDirectory="$(_AndroidLibraryFlatArchivesDirectory)"
-    ToolPath="$(Aapt2ToolPath)"
-    ToolExe="$(Aapt2ToolExe)">
-    <Output TaskParameter="CompiledResourceFlatArchives" ItemName="_UpdatedFlatArchives" />	
-  </Aapt2Compile>
-  <Touch Files="@(_UpdatedFlatArchives->'$(_AndroidLibraryFlatArchivesDirectory)\%(Filename).stamp')"
-    Condition=" '$(AndroidUseAapt2)' == 'True' And '@(_UpdatedFlatArchives)' != '' "
-    AlwaysCreate="True"
-  />
+  <CallTarget Targets="_FixupCustomViewsForAapt2" Condition=" '$(_AndroidUseAapt2)' == 'True'" />
```

Comment:
This one is in the middle, so yeah `<CallTarget/>` might be the only option here.

Reply:
I had to rework this anyway since `_FixupCustomViewsForAapt2` uses an item group that was created in the `_GenerateJavaStubs` target. So I've added a `_AfterGenerateJavaStubs` property group.